src/SmartComponents/modules/reducers.js

```diff
@@ -40,6 +41,17 @@ function filterCompareData(data, stateFilter, factFilter) {
         }
     }
 
+    if (sort === 'asc') {
+        filteredFacts.sort(function(a, b) {
+            return a.name > b.name;
+        });
+    }
+    else if (sort === 'desc') {
+        filteredFacts.sort(function(a, b) {
+            return b.name > a.name;
+        });
+    }
```

Comment:
I think this overloads the filtering function. Any data should be sortable, so splitting it out to a function just like filtering and pagination makes it more atomic and easier to test.